_data/utilities.yml

```diff
@@ -73,9 +73,8 @@ websites:
       tfa: No
 
     - name: Dominion
-      url: https://www.dom.com/
-      twitter: DomVAPower
-      facebook: dominionvirginiapower
+      url: https://www.dominionenergy.com/
+      twitter: dominionenergy
```

Comment:
It appears that this account is solely used as a corporate news Twitter, and it doesn't look like they reply to tweets. Should we use a different account?